settings-sample.sh

```diff
@@ -0,0 +1,6 @@
+# Please copy this file to settings.sh, grant it executable permissions and modify as needed.
```

Comment:
we you change the extension to `.conf`

Reply:
To keep things easy it's simply reading  the settings file as another bash script. I can change the extension but then that might be confusing to others. Up to you.